test/e2eprovider/server/server.go

```diff
@@ -0,0 +1,144 @@
+package e2eprovider
```

Comment:
We should add build tags to these files `// +build e2e` and update go build to build with the e2e tag

Reply:
done